lib/locales/en.ts

```diff
@@ -632,6 +651,9 @@ export const locale: Locale = {
     landry_of_paris_bishop: 'Saint Landry of Paris, Bishop',
     laserian_of_leighlin_bishop: 'Saint Laserian, Bishop',
     laura_vicuna_virgin: 'Blessed Laura Vicuña, Virgin',
+    laud_of_coutances_bishop: 'Saint Laud of Coutances, Bishop',
+    laud_of_coutances_bishop_patron_of_the_diocese_of_coutances:
+      'Saint Laud, Bishop, Patron of the Diocese of Coutances',
```

Comment:
We should be more consistent. Use either of the following suggestions.

```suggestion
    laud_of_coutances_bishop: 'Saint Laud, Bishop',
    laud_of_coutances_bishop_patron_of_the_diocese_of_coutances:
      'Saint Laud, Bishop, Patron of the Diocese of Coutances',
```

```suggestion
    laud_of_coutances_bishop: 'Saint Laud of Coutances, Bishop',
    laud_of_coutances_bishop_patron_of_the_diocese_of_coutances:
      'Saint Laud of Coutances, Bishop, Patron of the Diocese of Coutances',
```

Reply:
I think the first one is fine. If there's any doubt about which Saint Laud it is (if there are multiple), we can always check the corresponding ID.